mp/src/game/shared/momentum/weapon/weapon_mom_concgrenade.h

```diff
@@ -40,6 +40,7 @@ class CMomentumConcGrenade : public CWeaponBase
 
   protected:
     CNetworkVar(bool, m_bPrimed);      // Set to true when the pin has been pulled but the grenade hasn't been thrown yet.
+    CNetworkVar(bool, m_bRepress);     // Require a release and repress of +attack before a new grenade can be primed
```

Comment:
You should probably rename this to be a bit better:
```suggestion
    CNetworkVar(bool, m_bNeedsRepress);     // Require a release and repress of +attack before a new grenade can be primed
```

Reply:
Also, I wonder if weapon code has something like this, like `CanFire` or something... would need to dig deeper into code when not on my phone.